tests/integration/it_auth_test.py

```diff
@@ -63,27 +64,25 @@ def test_activate(client):
 
 
 """
-    @todo #370:30min Forgot password routine. At the moment we are showing the
-     e-mail from which we sent the new password link at the forgot_password_post 
-     page. We should mask this e-mail somehow (like p******o@gmail.com) so 
-     just the user get a hint to where the email was sent. After implementing 
-     the masking correct it in the test below
     @todo #388:30min Forgot password routine. Test a forgot password logic: 
      create an random password, Login with it, see it works.
      change it for the user with the received e-mail, find it changed.
 """
 
 
-@pytest.mark.skip(reason="Can't inject mock user base")
-def test_forgot_password_post(client):
-    email = "test@mail.com"
+@pytest.mark.parametrize("email,masked_email", (
+    ("john.smith@gmail.com", "joh*******@gmail.com"),
+    ("mrgreen@yahoo.com", "mrg****@yahoo.com"),
+))
+def test_forgot_password_post(client, email, masked_email):
+    employee = factories.EmployeeFactory(email=email)
     response = client.post(flask.url_for("auth.forgot_password"), data={
-        "email": email
-    })
+        "email": employee.email})
```

Comment:
@vryazanov why ``employee.email`` and not ``email``? What do we do with the ``email`` provided in the ``@pytest.mark.peremeterize``? :D

Reply:
@amihaiemil `email` provided in `parametrize` mark goes to `factories.EmployeeFactory(email=email)`. so there are no difference in using `"email": employee.email` or `"email": email` in this case